### PR TITLE
Polygon painting

### PR DESCRIPTION
**Pull request by RemiCecchinato**
@finetjul

### CODE REVIEW

**Comment by finetjul** [comment]
Warning, 1 commit has "SplineWidget" as feature, and the other "vtkSplineWidget", rename the 2nd one to "feat(SplineWidget)"

File: Sources/Widgets/Widgets3D/SplineWidget/index.js [hunk 1]
```diff
@@ -62,6 +62,7 @@ const DEFAULT_VALUES = {
     key: 'Shift',
     status: 'down',
   },
+  defaultCursor: 'pointer',
```
Comment (finetjul) [inline Sources/Widgets/Widgets3D/SplineWidget/index.js]: how about having that feature for ALL widgets ?

Comment (RemiCecchinato) [reply Sources/Widgets/Widgets3D/SplineWidget/index.js]: Not all widgets change the cursor and some widgets might want some more complex systems than just defaultCursor/grabbingCursor (eg. IpmlicitPlaneWidget uses 4 different cursors).

Comment (finetjul) [reply Sources/Widgets/Widgets3D/SplineWidget/index.js]: @floryst what do you think ?

Comment (floryst) [reply Sources/Widgets/Widgets3D/SplineWidget/index.js]: Exposing an API to modify the default/dragging/etc. cursors seems like a reasonable idea. However, if this API is added to the widget superclass, then there has to be some way of accommodating various cursor states (e.g. if a widget has special cursors for dragging edges vs corners, etc.) Right now I think it'll be easier for users who want to change cursor state to subclass the widget.

Also, do you want to add a `macro.setGet` in extend() for `defaultCursor`?

Comment (RemiCecchinato) [reply Sources/Widgets/Widgets3D/SplineWidget/index.js]: Thank you @floryst .

**Pull request merged**